client/app/previous-pres/previousPres.controller.js

```diff
@@ -8,6 +9,15 @@ angular.module('spkr.previous-pres', ['ngRoute'])
       }
     }, true);
 
+        $scope.class = "glyphicon-plus-sign";
+    
+    $scope.changeClass = function(){
+        if ($scope.class === "glyphicon-plus-sign")
+            $scope.class = "glyphicon-minus-sign";
+         else
+            $scope.class = "glyphicon-plus-sign";
```

Comment:
if statement on line 5 contains brackets, but this one doesn't. Ideally these would be made consistent; brackets are advisable. Optionally, line 16 could be stuck on the end of line 15, and line 18 on the end of line 17.